### PR TITLE
Security Document categories/upload on Admin view

### PR DESCRIPTION
**Pull request by mmumby**
This PR strayed a little bit from the original ticket. 

Here's whats happening: 
**Securities**
After some deliberation, we decided that a "placeholder bond record" was not necessary to accomplish the following.
We needed a spot to upload documents related to securities on the NoW admin view:
- added a security document categories
- Filtered the above categories and allowed users to upload documents
Screen Shot 2020-08-13 at 5 15 18 PM
Screen Shot 2020-08-13 at 5 16 28 PM

**Government Documents**
- Since this was very similar to Securities above and very little effort on my part, added the initial pass for this section (Will likely have to add more categories in the future)
Screen Shot 2020-08-13 at 5 20 45 PM
Screen Shot 2020-08-13 at 5 20 57 PM

**Securities again**
- Move "Securities Total" and "Security Received Date" To the admin view according to the designs
- Updated where the data is saved - moved from NoW to permit amendment so it can be carried forward when the permit is issued. 
- You must have initiated the draft permit process before adding the security details
Screen Shot 2020-08-13 at 5 26 59 PM
Screen Shot 2020-08-13 at 5 27 47 PM

### CODE REVIEW

File: services/core-api/app/api/mines/permits/permit_amendment/models/permit_amendment.py [hunk 1]
```diff
@@ -22,31 +22,44 @@ class PermitAmendment(AuditMixin, Base):
...
-    deleted_ind = db.Column(db.Boolean, nullable=False, server_default=FetchedValue())
+    deleted_ind = db.Column(db.Boolean,
+                            nullable=False,
+                            server_default=FetchedValue())
     lead_inspector_title = db.Column(db.String, nullable=True)
     regional_office = db.Column(db.String, nullable=True)
 
     permit_amendment_status = db.relationship('PermitAmendmentStatusCode')
-    permit_amendment_status_description = association_proxy('permit_amendment_status',
-                                                            'description')
```
Comment (Jsyro) [inline services/core-api/app/api/mines/permits/permit_amendment/models/permit_amendment.py:42]: What does your .vscode/settings.json file "python.formatting.yapfArgs" key look like? This looks like competing formatters.

File: services/core-api/app/api/mines/response_models.py [hunk 2]
```diff
@@ -105,6 +109,7 @@ def format(self, value):
         'issue_date': fields.DateTime(dt_format='iso8601'),
         'authorization_end_date': fields.DateTime(dt_format='iso8601'),
         'security_total': fields.Fixed(description='Currency', decimals=2),
+        'security_received_date': fields.DateTime(dt_format='iso8601'),
```
Comment (Jsyro) [inline services/core-api/app/api/mines/response_models.py:112]: we should have been able to get away from this iso8601 flag. Do we need this parameter?

File: services/core-api/app/api/mines/response_models.py [hunk 3]
```diff
@@ -8,6 +8,10 @@ class DateTime(fields.Raw):
     def format(self, value):
         return value.strftime("%Y-%m-%d %H:%M") if value else None
 
+class Date(fields.Raw):
+    def format(self, value):
+        return value.strftime("%Y-%m-%d") if value else None
+
```
Comment (Jsyro) [inline services/core-api/app/api/mines/response_models.py:14]: is this used?

File: services/core-api/app/api/mines/permits/permit_amendment/models/permit_amendment.py (hunk 1 above)
Comment (mmumby) [reply services/core-api/app/api/mines/permits/permit_amendment/models/permit_amendment.py:42]: Apparently it didn't exist and the mds one was not taking effect, I have updated it to be the same as mds. I'll resave all these files and push up the change

```
"python.formatting.yapfArgs": [
    "--style={column_limit:100, spaces_before_comment:'15,30,50', SPLIT_BEFORE_EXPRESSION_AFTER_OPENING_PAREN:TRUE}"
  ],
```

File: services/core-api/app/api/securities/resources/bond.py [hunk 4]
```diff
@@ -127,11 +128,16 @@ def put(self, bond_guid):
...
-            raise BadRequest('This bond is already associated with this permit.')
-        if bond.permit.mine_guid not in [m.mine_guid for m in permit._all_mines]:
-            raise BadRequest('You can only transfer to a permit on the same mine.')
+            raise BadRequest(
+                'This bond is already associated with this permit.')
+        if bond.permit.mine_guid not in [
+                m.mine_guid for m in permit._all_mines
+        ]:
+            raise BadRequest(
+                'You can only transfer to a permit on the same mine.')
```
Comment (Jsyro) [inline services/core-api/app/api/securities/resources/bond.py:140]: I don't see any functional changes in this file?

File: services/core-api/app/api/mines/response_models.py (hunk 2 above)
Comment (mmumby) [reply services/core-api/app/api/mines/response_models.py:112]: no, I added it for consistency with the other date fields. should it just be `fields.DateTime()`?

File: services/core-api/app/api/mines/response_models.py (hunk 3 above)
Comment (mmumby) [reply services/core-api/app/api/mines/response_models.py:14]: not anymore, I'll remove it.

File: services/core-api/app/api/securities/resources/bond.py (hunk 4 above)
Comment (mmumby) [reply services/core-api/app/api/securities/resources/bond.py]: When I started the branch, I made changes to the bond model/resource and then removed all changes/pivoted the feature. This file is just formatting changes now which will hopefully be resolved once I update the formatting issue.

File: services/core-api/app/api/mines/response_models.py (hunk 2 above)
Comment (Jsyro) [reply services/core-api/app/api/mines/response_models.py:107]: I think that should work just as well yes.

**Review by Jsyro — DISMISSED** [review]
Looks great, thanks for fixing the formatting issues.

**Pull request merged**